src/app/pages/restaurant-details-page/restaurant-details-page.component.html

```diff
@@ -1,4 +1,5 @@
 <div class="restaurant-page">
+  <navigation-bar></navigation-bar>
```

Comment:
Add navigation bar in app.component.html before the router-outlet, like that you will only have to add in one place